packages/host/tests/integration/components/operator-mode-test.gts

```diff
@@ -2575,14 +2576,25 @@ module('Integration | operator-mode', function (hooks) {
         clientX: itemRect.left + 1,
         clientY: itemRect.top + 1,
       });
+
+      let firstStackItemHeaderRect = document
+        .querySelector('[data-test-operator-mode-stack="0"] header')!
+        .getBoundingClientRect();
+      let firstStackItemPaddingTop = getComputedStyle(
+        document.querySelector('[data-test-operator-mode-stack="0"]')!,
+      )
+        .getPropertyValue('padding-top')
+        .replace('px', '');
+      let marginTop =
+        firstStackItemHeaderRect.height + Number(firstStackItemPaddingTop);
       await triggerEvent(document, 'mousemove', {
         clientX: targetRect.left + targetRect.width / 2,
-        clientY: targetRect.top - 50,
+        clientY: targetRect.top - marginTop,
       });
 
       await triggerEvent(itemElement, 'mouseup', {
         clientX: targetRect.left + targetRect.width / 2,
-        clientY: targetRect.top - 50,
+        clientY: targetRect.top - marginTop,
```

Comment:
Changes to this test are unrelated to this PR. However, they fix the failing test on main.